trav/main.py

```diff
@@ -1,12 +1,34 @@
 import os
+import requests
+
+TRAVIS_BASE_URL = 'https://api.travis-ci.org/'
+
+
+class SvgRequestFailed(Exception):
+    pass
+
 
 def var_is_true(var):
     if os.environ.get(var) and os.environ.get(var).lower() == 'true':
         return True
     else:
         return False
 
+
 class Travis(object):
+    @staticmethod
+    def get_svg_from_badge_url(url):
+        svg = requests.get(url)
+        if svg.status_code != 200:
+            raise SvgRequestFailed("Unable to complete the SVG request")
+        return svg
+
+    @staticmethod
+    def get_travis_badge_url(group, repo, branch):
+        return TRAVIS_BASE_URL + '{group}/{repo}.svg?branch={branch}'.format(
```

Comment:
Please have a  `TRAVIS_BADGE_URL  = TRAVIS_BASE_URL + '{group}/{repo}.svg?branch={branch}'` globally and then use it here.